qiskit/circuit/__init__.py

```diff
@@ -56,6 +56,55 @@
 Supplementary Information
 =========================
 
+.. dropdown:: Quantum Circuit with conditionals
+   :animate: fade-in-slide-down
+
+   When building a quantum circuit, there can be interest in applying a certain gate only
+   if a classical register has a specific value. This can be done with the
+   :meth:`InstructionSet.c_if` method.
+
+   In the following example, we start with a single-qubit circuit formed by only a Hadamard gate,
```

Comment:
Did you want to link to the `HGate` class here maybe like:

```suggestion
   In the following example, we start with a single-qubit circuit formed by only a Hadamard gate (:class:`~.HGate`),
```